Gruntfile.js

```diff
@@ -39,6 +39,10 @@ module.exports = function(grunt) {
             },
             dist_progress: {
                 src: [
+                    'public/javascripts/Admin/src/*.js',
+                    'public/javascripts/SVValidate/src/util/*.js',
```

Comment:
It looks like we probably only need the `PanoProperties.js` file in here, not all of them, so let's just include that one for now!

Reply:
I tried removing some of the files but looks like all of them need to be included otherwise the map doesn't render correctly

Reply:
Thanks for checking!